docs/getting_started/tutorials/index.md

```diff
@@ -1,5 +1,7 @@
 # Tutorials
 
+Many BIDS tutorials have been created by the greater BIDS community. We aim to add high more high-quality tutorials to this list as new tutorials are created and the maintainers become aware of them. If there are good tutorials that are missing from this list, please feel free to put in a request on github for them to be added to this website.  
```

Comment:
- [ ] Add a link to Where they're supposed to add the request on github. like maybe the Issues page?   
```suggestion
Many BIDS tutorials have been created by the greater BIDS community and a selection are linked here. We aim to add more high-quality tutorials to this list as new tutorials are created and the maintainers become aware of them. Let us know about tutorials that are missing from this list -- add a request on github to add them to this website.  
```
- fixing what's ungrammatical in the above, clarifying that we're linking to some (not all).
- removing words so that this reads smoother
- [ ] please tidy the last sentence to more directly Invite info on new tutorials via the GitHub Issues(?) page (and again make sure to link to it).

Reply:
made edits per suggestions. :)